htsinfer/infer_read_orientation.py

```diff
@@ -22,5 +21,45 @@ def infer(
         LIBTYPE string according to Salmon documentation, cf.
         https://salmon.readthedocs.io/en/latest/library_type.html
     """
+    #create .tmp directory in "inputDir" and retuns .tmp path
+    tmpDir = mkTmpDir(inputDir)
+    
+    # call functions of issues #25, 26, 27 and 34
+    
+    # delete .tmp
+    rmTmpDir(tmpDir)
+        
     # implement logic
     return "U"
+    
+
+def mkTmpDir(inputDir):
+    """Function to create a temporary directory for the storage of 
+    temporary data."""
+  
+    import os
+
+    # define the name of the directory to be created
+    tmpPath = inputDir + "/.tmp"
```

Comment:
This is not a very good name for the temporary directory because the chance of collisions is quite high and we don't want to risk overwriting data. There is a standard/builtin module `tempfile` in Python for such cases and it's function `mkdtemp()` is specifically made for our purpose. So you add `from tempfile import mkdtemp` to the top of the file and then create a directory the directory under `input_dir` with `mkdtemp(dir=input_dir)`. This will both generate a name _and_ create the input directory and then return the name of the directory.

In fact you can just simply return it like so:

```py
return mkdtemp(dir=input_dir)
``` 

This is all you need to do in this function. However, since your function will then just be a single line, it doesn't make sense to have a function for this at all, so you can just add this call to the main `infer()` method:

```py
tmp_dir = mkdtemp(dir=__path__)
```

This will create the temporary directory in the directory where the current directory is located.